src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,22 +1,24 @@
 package mate.academy.service;
 
-import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
-    public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+    public User login(String login, String password) {
+        logger.info("login() was called. Params: login = {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.error("Username or password are incorrect. Login: " + login);
```

Comment:
Do not delete AuthenticationException throwing. We are adding logging without logic change.